src/sql/parts/notebook/models/notebookModel.ts

```diff
@@ -253,10 +260,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		try {
 			this._trustedMode = isTrusted;
 			let contents = null;
-			if (this._notebookOptions.notebookUri.scheme !== Schemas.untitled) {
-				// TODO: separate ContentManager from NotebookManager
-				contents = await this.notebookManagers[0].contentManager.getNotebookContents(this._notebookOptions.notebookUri);
+			let notebookEditorModel: NotebookEditorModel;
+
+			if (this._notebookParams && this._notebookParams.input) {
+				notebookEditorModel = await this._notebookParams.input.resolve();
+				let localContentManager = new LocalContentManager();
```

Comment:
should this always be local content manager? We'll have a remote content manager at some point, right?